_pytest/fixtures.py

```diff
@@ -835,8 +848,10 @@ def __init__(self, scope, params, autouse=False, ids=None, name=None):
     def __call__(self, function):
         if isclass(function):
             raise ValueError(
-                "class fixtures not supported (may be in the future)")
-        function._pytestfixturefunction = self
+                    "class fixtures not supported (may be in the future)")
+        if getattr(function, "_pytestfixturefunction", False):
+            function = copy_func(function)
+
```

Comment:
the assigment was reoved, as such this is broken

from my pov we should not copy the function but warn about the mistake reserving the right to fail later on

Reply:
I agree that copying the function is dangerous. I think we can target `features` and raise an error if `@pytest.fixture` is being applied more than once to the same function.

It is clearly a problem and if it is working is mostly by accident.